### ISSUE TITLE
`ConditionalBindingCascadeRule` does not check `where` clause­

### ISSUE THREAD

**Issue by norio-nomura** [realm/SwiftLint#347]
`let` is needed after `where` clause­.
So, `ConditionalBindingCascadeRule` should not trigger on following example:
```swift
"guard let a = b where a, let c = d {"
```

**Comment by jpsim** [comment]
Addressed in #348.
